src/array.ts

```diff
@@ -132,6 +137,11 @@ declare global {
     min(this: readonly number[] | readonly Date[]): T;
     min<R extends number | Date>(f: (el: T, index: number, array: readonly T[]) => R): R;
     count(f: (el: T, index: number, array: readonly T[]) => boolean): number;
+    /**
+     * Compares every element in the array to each other and returns true if they all match `fn(el)` and
+     * false otherwise. Shorthand for `array.every((el, _, [first]) => el.some.id === first.some.id)`
+     */
+    unanimous(f: (el: T) => unknown): boolean;
```

Comment:
@TylerR909 / @zgavin wdyt about working `every` into this method name? It's so similar to `every`, that something like `.everyHasSame(... => ...)` seems more "ah I get it" to me than the "admittedly cute but I think it's slightly too opaque" `unanimous`.

Reply:
I agree I'm not crazy about the name.